gui/control_main.py

```diff
@@ -4498,9 +4498,12 @@ def refreshCollectionParams(self, selectedSampleRequest, validate_hdf5=True):
         energy_s = str(daq_utils.wave2energy(reqObj["wavelength"], digits=6))
         dist_s = str(reqObj["detDist"])
         self.detDistMotorEntry.getEntry().setText(str(dist_s))
-        self.dataPathGB.setFilePrefix_ledit(str(reqObj["file_prefix"]))
-        self.dataPathGB.setBasePath_ledit(str(reqObj["basePath"]))
-        self.dataPathGB.setDataPath_ledit(str(reqObj["directory"]))
+        # self.dataPathGB.setFilePrefix_ledit(str(reqObj["file_prefix"]))
+        # self.dataPathGB.setBasePath_ledit(str(reqObj["basePath"]))
+        # self.dataPathGB.setDataPath_ledit(str(reqObj["directory"]))
```

Comment:
If these are no longer required, please remove entirely.

Reply:
Addressed in https://github.com/NSLS-II/lsdc/pull/348/commits/0d74496d71ff9d936f82847f08a850e6228d28e8